cognite/v05/depthseries.py

```diff
@@ -0,0 +1,310 @@
+# -*- coding: utf-8 -*-
+"""Depthseries Module
+
+This module mirrors the Timeseries API, but handles pairs of timeseries to emulate depth(or other indexed) data series. It allows you to fetch data from the api and output it in various formats.
```

Comment:
Could/should it be called something other than depthseries, since it indeed can be something else? TwoDimensionalTimeseries or something?

Reply:
I'm not sure what I think here.. Could/should it be extended so that it can handle timeseries with N different values (not just two)?

Reply:
As it is now it is a depth series only, as we assume a unit of m and name the index accordingly. Its not really a timeseries with 2 different values though either, it has 1 value and 1 index, but the index in this case is not time. 

The trajectory work we will do next will be 2 or more data series, and AFAIK still depth as an index. Not sure yet how much that will just be an extension of this though.

Reply:
Let's keep this as depthseries then, and refactor if that makes sense when we look at trajectories. Pretty sure we'll be the only ones using this anyway :)